.flake8

```diff
@@ -1,2 +0,0 @@
-[flake8]
-max-line-length = 120
```

Comment:
Also, in setup.py's `requirements_dev` need to drop flake8, add pre-commit. 

[Long-term, I'd really like to modernize this project's packaging, which would eliminate a lot of setup.py. But that's a whole separate topic.]

Reply:
I updated requirements_dev in setup.py